widget/embedded/src/store/AppStore.tsx

```diff
@@ -28,9 +29,11 @@ export function useAppStore() {
 
 export function AppStoreProvider(props: AppStoreProviderProps) {
   const appStoreRef = useRef<AppStore>();
+  const [searchParams] = useSearchParams();
+  const liquiditySources = searchParams.get('liquiditySources')?.split(',');
 
   if (!appStoreRef.current) {
-    appStoreRef.current = createAppStore(props.config);
+    appStoreRef.current = createAppStore(props.config, { liquiditySources });
```

Comment:
What do you think about not getting this directly here? Our approach for `config` for is more appropriate here as well. a method for updating the config (`liquiditySources` here).

Reply:
```
campaignMode: {
...
}
updateCampaingData: () => ..
```